### PR TITLE
Change API version to v1alpha1

### PR DESCRIPTION
**Pull request by xing-yang**
This PR made the following changes:
- changes plugin CRD version from v1 to v1alpha1
- changes API group from `veleroplugin.io`  to `datamover.cnsdp.vmware.com`
- changes API group from `backupdriver.io` to `backupdriver.cnsdp.vmware.com`
- Updates apiextensions.k8s.io/v1beta1 to apiextensions.k8s.io/v1
- Updates controller-gen from v0.3.0 to v0.4.0

### CODE REVIEW

**Review by lintongj — APPROVED** [review]
/lgtm 

```
Updates apiextensions.k8s.io/v1beta1 to apiextensions.k8s.io/v1
```
Just curious why do we make this change.

File: README.md [hunk 1]
```diff
@@ -72,24 +72,24 @@ background and may take a significant amount of time to complete.
 
 ## Monitoring data upload progress
 
-For each volume snapshot that is uploaded to S3, an uploads.veleroplugin.io customer resource is generated.  These records contain the current state of an upload request.  You can list out current requests with
+For each volume snapshot that is uploaded to S3, an uploads.cnsdp.vmware.com customer resource is generated.  These records contain the current state of an upload request.  You can list out current requests with
```
Comment (swgupta-vmw) [inline README.md:75]: Should this be uploads.datamover.cnsdp.vmware.com?

File: README.md [hunk 2]
```diff
@@ -72,24 +72,24 @@ background and may take a significant amount of time to complete.
...
 
 ```bash
-kubectl get -n <velero namespace> uploads.veleroplugin.io -o yaml
+kubectl get -n <velero namespace> uploads.cnsdp.vmware.com -o yaml
```
Comment (swgupta-vmw) [inline README.md:78]: uploads.datamover.cnsdp.vmware.com

File: README.md [hunk 3]
```diff
@@ -72,24 +72,24 @@ background and may take a significant amount of time to complete.
...
+- apiVersion: cnsdp.vmware.com/v1
   kind: Upload
   metadata:
     creationTimestamp: "2020-03-16T21:12:48Z"
     generation: 1
     name: upload-bcc6e06c-8d2f-4e19-b157-0dbd1ef9fcb2
     namespace: velero
     resourceVersion: "2173887"
-    selfLink: /apis/veleroplugin.io/v1/namespaces/velero/uploads/upload-bcc6e06c-8d2f-4e19-b157-0dbd1ef9fcb2
+    selfLink: /apis/cnsdp.vmware.com/v1/namespaces/velero/uploads/upload-bcc6e06c-8d2f-4e19-b157-0dbd1ef9fcb2
```
Comment (swgupta-vmw) [inline README.md:92]: uploads.datamover.cnsdp.vmware.com

**Review by swgupta-vmw — APPROVED** [review]

**Comment by xing-yang** [comment]
> `Updates apiextensions.k8s.io/v1beta1 to apiextensions.k8s.io/v1`
> Just curious why do we make this change.

This change is not required but `apiextensions.k8s.io/v1beta1` is deprecated.

File: README.md (hunk 3 above)
Comment (xing-yang) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (xing-yang) [reply README.md]: Done

File: README.md (hunk 1 above)
Comment (xing-yang) [reply README.md]: Done

**Comment by xing-yang** [comment]
Tested backup/restore on my Vanilla setup with this fix and it passed.
Merging it now.

**Pull request merged**